CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @axelar-network/axelar-cgp-sui
 
```

Comment:
Not sure if the release action will work on this custom branch instead of the one I mentioned